pkg/syncer/pv_to_backingdiskobjectid_mapping.go

```diff
@@ -47,7 +47,21 @@ func csiGetPVtoBackingDiskObjectIdMapping(ctx context.Context, k8sclient clients
 			string(cnstypes.QuerySelectionNameTypeVolumeType),
 		},
 	}
-	queryAllResult, err := metadataSyncer.volumeManager.QueryAllVolume(ctx, queryFilter, querySelection)
```

Comment:
In the case of multi-vCenter can we connect to both vCenters using volume managers from both VCs?